helix-common/src/main/java/org/apache/helix/SystemPropertyKeys.java

```diff
@@ -34,6 +34,9 @@ public class SystemPropertyKeys {
   // Task Driver
   public static final String TASK_CONFIG_LIMITATION = "helixTask.configsLimitation";
 
+  // Task executor threadpool reset timeout in ms
+  public static final String TASK_THREADPOOL_RESET_TIMEOUT = "helixTask.threadpool.reset.timeout";
```

Comment:
nit - suggestion - helixTask.threadpool.resetTimeout (camel casing is done for others)

Reply:
Done, thanks